exp-models/addon/components/jam-auth/component.js

```diff
@@ -11,6 +11,11 @@ export default Ember.Component.extend({
     password: null,
 
     authenticating: false,
+    invalidAuth: false,
+
+    allowLogin: Ember.computed('username', 'password', 'authenticating', 'invalidAuth', function() {
```

Comment:
In the template usage, we disable the login button if this is true.

...So really, should this be named `disableLogin`?